hibernate-envers/src/main/java/org/hibernate/envers/internal/entities/mapper/relation/MiddleIdData.java

```diff
@@ -26,7 +26,7 @@ public MiddleIdData(
 			AuditEntitiesConfiguration verEntCfg, IdMappingData mappingData, String prefix,
 			String entityName, boolean audited) {
 		this.originalMapper = mappingData.getIdMapper();
-		this.prefixedMapper = mappingData.getIdMapper().prefixMappedProperties( prefix );
+		this.prefixedMapper = prefix == null ? null : mappingData.getIdMapper().prefixMappedProperties( prefix );
```

Comment:
as for the previous comment I'm not sure why this is needed, I tried to change it back to `this.prefixedMapper = mappingData.getIdMapper().prefixMappedProperties( prefix );` and all the tests passed